File: COP 3330/BlackBoxOfChaos/TestCase01.java
// Sean Szumlanski
// COP 3330, Spring 2023

// ================================
// BlackBoxOfChaos: TestCase01.java
// ================================
// A brief test case to help ensure you've implemented the difficultyRating()
// and hoursSpent() methods correctly.
//
// For detailed compilation and testing instructions, see the assignment PDF.


public class TestCase01
{
	public static void main(String [] args)
	{
		double difficulty = BlackBoxOfChaos.difficultyRating();
		System.out.println((difficulty < 1.0 || difficulty > 5.0) ? "oh no :(" : "Hooray!");

		double hours = BlackBoxOfChaos.hoursSpent();
		System.out.println((hours <= 0.0) ? "oh no :(" : "Hooray!");
	}
}

File: COP 3502/StringMuncher/testcase01.c
// Sean Szumlanski
// COP 3502, Spring 2023

// ===========================
// StringMuncher: testcase01.c
// ===========================
// This test case is a simple boundary check to make sure the difficultyRating()
// function is implemented correctly.


#include <stdio.h>
#include "StringMuncher.h"

int main(void)
{
	if (difficultyRating() < 1.0 || difficultyRating() > 5.0)
		printf("fail whale :(\n");
	else
		printf("Hooray!\n");

	return 0;
}
